### PR TITLE
Notification side sheet redesign

### PR DESCRIPTION
**Pull request by ingridklepsvik**
Added tabs to notification side sheet. 

feat: changed notification side sheet

### CODE REVIEW

File: src/components/general/NotificationsSideSheet/index.tsx [hunk 1]
```diff
@@ -13,59 +15,31 @@ type NotificationsSideSheetProps = {
...
 type NotificationsSideSheetContentProps = {
+    settings: React.ReactElement | null;
+    showSettings?: boolean;
     readNotifications: NotificationCard[];
     unReadNotifications: NotificationCard[];
     isFetchingReadNotifications: boolean;
     isFetchingUnReadNotifications: boolean;
 };
 
 const NotificationsSideSheetContent: React.FC<NotificationsSideSheetContentProps> = ({
```
Comment (odinr) [inline src/components/general/NotificationsSideSheet/index.tsx:26]: this file contains two components, there should only be one/file, split into 2 files

File: src/components/general/NotificationsSideSheet/index.tsx [hunk 2]
```diff
@@ -78,6 +52,70 @@ const NotificationsSideSheet: React.FC<NotificationsSideSheetProps> = ({
...
+                                        <Button
+                                            onClick={onReadAll}
+                                            disabled={unReadNotifications.length <= 0}
+                                        >
+                                            {isMarkingNotifications ? <Spinner inline /> : 'Mark all as read'}
+                                        </Button>
+                                    <NotificationDateDivisions notifications={unReadNotifications} />
+                                    </>
+                                ))}
+                    </div>
```
Comment (odinr) [inline src/components/general/NotificationsSideSheet/index.tsx:84]: too much logic inside render method, either refactor to callback/memo or create a new component(s)

**Review by odinr — CHANGES_REQUESTED** [review]

File: src/components/general/NotificationsSideSheet/index.tsx (hunk 1 above)
Comment (ingridklepsvik) [reply src/components/general/NotificationsSideSheet/index.tsx]: split the two components into two files

File: src/components/general/NotificationsSideSheet/index.tsx (hunk 2 above)
Comment (ingridklepsvik) [reply src/components/general/NotificationsSideSheet/index.tsx]: Removed some logic from the render method and refactored to memo

**Review by odinr — APPROVED** [review]

**Pull request merged**